nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/utils/DecodeUtil.scala

```diff
@@ -53,6 +53,17 @@ object DecodeUtil {
     (encodeDataframe, encodeId)
   }
 
+  def convertEncodeId2StringId(dataframe: DataFrame, encodeId: DataFrame): DataFrame = {
```

Comment:
What's the difference between convertEncodeId3StringId and converAlgoId2StringId

Reply:
I re-examined and tested the code, and found that it is indeed the same. 
It should be a misjudgment caused by the problem of receiving the encodeId parameter of the degree_static algorithm. 
Let me resubmit the code that solves the algorithm encodeId configuration parameter passing problem.